src/client-python/reportclient/internal/bz_connection.py

```diff
@@ -26,6 +26,8 @@
 from typing import Any, Dict, List, Optional
 
 import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
```

Comment:
urllib3 is not listed as a dependency in the spec file. Should it be added?

Reply:
Good catch. `urllib3` is a dependency of [requests](https://github.com/psf/requests/blob/main/setup.py#L64) and that's probably why I haven't seen it explicitly listed in requirements very often.
But I actually wanted to use the built-in version that comes with Python itself... let me check what can be done here.

Reply:
Yeah, it would be better to explicitly list `urllib3` as a a dependency: https://bugzilla.redhat.com/show_bug.cgi?id=1785666

Reply:
I added both `requests` and `urllib3` to the spec file.